src/quick-start.md

```diff
@@ -58,23 +72,23 @@ spec:
 
 Overview of the attributes of the `PolicyServer` resource:
 
-* `image`: container image name
-* `replicas`: number of desired instances
-* `serviceAccountName` (optional): `policy-server` ServiceAccount. Namespace's `default` ServiceAccount will be used if nothing is provided
-* `env` (optional): `policy-server` environment variables
-* `annotations` (optional): `policy-server` annotations
+| Required | Placeholder         | Description    |
+|:--------:| ------------------- | ----------------------------- |
+| [x] | `image`  | The name of the container image |
+| [x] | `replicas`  | The number of desired instances |
+| [] | `serviceAccountName` | The name of the `ServiceAccount` to use for the `policy-server` deployement. If no value is provided, the `ServiceAccount` from the `default` namespace will be used |
+| [] | `env` | The list of environment variables |
+| [] | `annotations` | The list of annotations |
 
 Changing any of these attributes will lead to a rollout of the `policy-server` Deployment with the new configuration.
 
-## Kubewarden Policies
+### ClusterAdmissionPolicy
+
+The `ClusterAdmissionPolicy` resource is the core of the Kubewarden stack. This component defines how the policies are validated.
```

Comment:
```suggestion
The `ClusterAdmissionPolicy` resource is the core of the Kubewarden stack. This resource defines how the policies are evaluated.
```

Nit: policies are not validated, the new resources that get evaluated by the policies are.(well yes, policies are validated, but they normally get validated by testing them when writing them, and yes, they get validated by the policy-server when instantiating them. I just worry that this is more )